music21/analysis/discrete.py

```diff
@@ -568,33 +560,33 @@ def _bestKeyEnharmonic(self, pitchObj, mode, sStream=None):
         >>> ks._bestKeyEnharmonic(pitch.Pitch('f-'), 'major', s)
         <music21.pitch.Pitch E>
 
-        '''
+        """
         if pitchObj is None:
             return None
 
         # this does not yet seem necessary
         # if not done at init with ref stream, do now
-#         if self.sharpFlatCount is None:
-#             sharpFlatCount = self._getSharpFlatCount(sStream)
-#         else:
-#             sharpFlatCount = self.sharpFlatCount
-#
-#         if sharpFlatCount[0] > sharpFlatCount[1]:
-#             favor = 'sharp'
-#         elif sharpFlatCount[1] > sharpFlatCount[0]:
-#             favor = 'flat'
-#         else:
-#             favor = None
+        #         if self.sharpFlatCount is None:
```

Comment:
when indenting comments, the spaces on the other side of the `#` must be removed.